mutiny-core/src/nodemanager.rs

```diff
@@ -1060,6 +1063,79 @@ impl<S: MutinyStorage> NodeManager<S> {
         })
     }
 
+    pub async fn send_payjoin(
+        &self,
+        uri: PjUri<'_>,
+        amount: u64,
+        labels: Vec<String>,
+        fee_rate: Option<f32>,
+    ) -> Result<Txid, MutinyError> {
+        let address = Address::from_str(&uri.address.to_string())
+            .map_err(|_| MutinyError::PayjoinConfigError)?;
+        let original_psbt = self.wallet.create_signed_psbt(address, amount, fee_rate)?;
+
+        let payout_scripts = std::iter::once(uri.address.script_pubkey());
+        let fee_rate = if let Some(rate) = fee_rate {
+            FeeRate::from_sat_per_vb(rate)
+        } else {
+            let sat_per_kwu = self
+                .fee_estimator
+                .get_est_sat_per_1000_weight(ConfirmationTarget::Normal);
```

Comment:
I ran into some type errors when trying to link for tests, it seems to think there is no such thing as `ConfirmationTarget::Normal`

Reply:
might have broke after the upgrade to LDK 0.0.118

Reply:
there's now `fee_estimator.get_normal_fee_rate()`

Reply:
tACK `.get_normal_fee_rate()` works